### DISCUSSION TITLE
Add WebdriverIO.Device ?

### DISCUSSION THREAD

**Discussion by DevDengChao** [webdriverio/webdriverio#7826] [General]
Since Appium also uses `wdio.remote()` to connect to Android/iOS devices, I think it is reasonable to replace `WebdriverIO.Browser` with `WebdriverIO.Device` and remain `WebdriverIO.Browser` as a type alias to keep backward compatibility.

**Comment by christian-bromann** [comment]
Sounds like a reasonable request. Any contributions are welcome!
